src/neptune/new/cli/sync.py

```diff
@@ -80,8 +81,9 @@ def sync_execution(
         container_id: UniqueId,
         container_type: ContainerType,
     ) -> None:
+        operation_storage = OperationStorage(str(execution_path))
```

Comment:
Why convert `Path` to `str`?

Reply:
Because `OperationStorage` requires string, but ok. I can make `OperationStorage` initializer more flexible.